packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { string, node } from 'prop-types';
+import styled from 'styled-components';
+import { C_WHITE } from '@bbc/psammead-styles/colours';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import { GEL_GROUP_2_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_LONG_PRIMER } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const GUIDANCE_BACKGROUND = 'rgba(34, 34, 34, 0.75)';
+
+const StyledGuidance = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  ${GEL_LONG_PRIMER};
+`;
+
+const GuidanceWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  background-color: ${GUIDANCE_BACKGROUND};
+  color: ${C_WHITE};
+  padding: ${GEL_SPACING};
```

Comment:
Suggestion: remove this global `padding` and let the children position themselves. This means we won't need to refactor if a future child needs to be padded differently.